systemtests/tests/always-incremental-consolidate/etc/bareos/bareos-dir.d/job/backup-bareos-fd.conf.in

```diff
@@ -7,4 +7,8 @@ Job {
   AlwaysIncrementalJobRetention = 1 seconds
   AlwaysIncrementalKeepNumber = 1
   AlwaysIncrementalMaxFullAge = 1 seconds
+  Run Script {
```

Comment:
Question: The job time start is took into account only after before script is completely done, is this instruction really help us to be on the safe side when time calculation is done ?

Reply:
What we need to take in consideration is the end time not start time. Shifting the start will definitely shift the end time.